pages/projects/projects-spacetimecube.tsx

```diff
@@ -214,6 +273,11 @@ const ProjectSpaceTimeCube: NextPage<Props> = ({
                 </Canvas>
               </CanvasStage>
             </div>
+            <div className="z-50 mr-4 mt-4 justify-self-start [grid-area:1/1]">
+              {selectedYear && (
+                <Bar height={250} options={options} data={data} />
+              )}
+            </div>
             <div className="z-50 mr-4 mt-4 justify-self-end [grid-area:1/1]">
               <SelectedProjectsDetails
                 selectedCountries={selectedCountries}
```

Comment:
```suggestion
            <div className="z-50 mr-4 mt-4 justify-self-end [grid-area:1/1]">
            {selectedYear && <SelectedYearDetails year={selectedYear} />}
              <SelectedProjectsDetails
                selectedCountries={selectedCountries}
```

I would suggest to move all code related to the barchart into a separate component (as it is not that much related to the spacetimecube). It would also move the barchart to the same side as the other details card to make the layout hopefully a bit more intuitiv: <img width="1421" alt="Screenshot 2023-11-14 at 08 54 19" src="https://github.com/jakoblistabarth/itc-atlas/assets/76778035/6f08f100-7d5e-4614-9167-e6b1e2840692">

Reply:
Ok.